src/radical/entk/execman/base/resource_manager.py

```diff
@@ -139,6 +139,20 @@ def shared_data(self, data_list):
         self._shared_data = data_list
 
 
+    @property
+    def outputs(self):
+        """
+        :getter:    list of files to be staged from remote after execution
+        """
+        return self._outputs
+
+
+    @shared_data.setter
```

Comment:
For readibility and consistency, should we separate getters and setters as done in `appmanager`? I would try not to make the code grow 'organically'.